operator/images/cluster-setup/content/bin/install.sh

```diff
@@ -154,8 +154,8 @@ install_clusters() {
 install_shared_manifests() {
   CREDENTIALS_DIR="$WORKSPACE_DIR/credentials"
 
-  # if [ "$(kubectl get secret -n tekton-chains signing-secrets --ignore-not-found -o json | jq -r ".immutable")" != "true" ]; then
-  #   kubectl apply -k "$CREDENTIALS_DIR/manifests/compute/tekton-chains"
+  # if [ "$(kubectl get secret -n openshift-pipelines signing-secrets --ignore-not-found -o json | jq -r ".immutable")" != "true" ]; then
```

Comment:
@ramessesii2 @Roming22 do we think we'll ever uncomment these lines and add them back?  If not, we should just delete this commented out check.

Reply:
I'm fine with that. I'll wait for @Roming22's thoughts on this.

Reply:
Let's go ahead and delete them.